src/templates/home-page.js

```diff
@@ -73,13 +74,14 @@ const BLMSolidarityModal = () => {
           </p>
         </main>
         <p style={{marginBottom:"1rem"}}>
-            Please consider donating to <a href="https://blacklivesmatter.com/" noreferrer>Black Lives Matter</a>, 
-            <a href="https://www.naacpldf.org/" noreferrer>The NAACP Legal Defense and Educational Fund</a>, 
-            <a href="https://eji.org/">The Equal Justice Initiative</a>, 
-            <a href="https://www.wetheprotesters.org/">We The Protesters and Campaign Zero</a>,
-            <a href="https://codecooperative.org/">Code Cooperative</a>,
-             and 
-            <a href="https://www.gofundme.com/f/georgefloyd"> George Floyd Memorial Fund</a>.
+            Please consider donating to <a href="https://bailfunds.github.io/">your local bail fund</a>,
```

Comment:
I noticed a tiny UI nit: We don't render spaces in between items on the list of links. If we want to maintain the multiline layout in the code (which definitely makes it easier to read), maybe we can add some non-breaking spaces with `&nbsp;` after each comma.